lib/bibliothecary/parsers/maven.rb

```diff
@@ -11,6 +11,8 @@ class Maven
       # "|    \\--- com.google.guava:guava:23.5-jre (*)"
       GRADLE_DEP_REGEX = /(\+---|\\---){1}/
 
+      MAVEN_DEP_REGEXT = /(\s+)([\w\.-]+)\:([\w\.-]+)\:([\w\.-]+)\:([\w\.-]+)\:([\w]+).+/
```

Comment:
My regex fu is weak but this is meant to match up each section of the dependency string so that can easily be set in the correct location in the parser.